### PR TITLE
all risk like validators documented EXCEPT paf

### PR DESCRIPTION
**Pull request by kfwilson**
Here's doc strings for all the risk like raw validators with the exception of paf - I want to wait for Michelle's PR to come in since it changes the validator quite a bit.

### CODE REVIEW

File: src/vivarium_inputs/validation/raw.py [hunk 1]
```diff
@@ -771,6 +853,37 @@ def validate_exposure_distribution_weights(data: pd.DataFrame, entity: Union[Ris
 
 def validate_relative_risk(data: pd.DataFrame, entity: Union[RiskFactor, CoverageGap], location_id: int,
                            exposure: pd.DataFrame, estimation_years: pd.Series)-> None:
+    """Check the standard set of validations on raw relative risk data for
+    entity, replacing the age ids check with a custom check based on the age
+    groups present in the exposure data for this entity. Check age and sex ids
+    on data grouped by cause, mortality, morbidity, and parameter.Only sex r
```
Comment (collijk) [inline src/vivarium_inputs/validation/raw.py]: Weird line wrap

File: src/vivarium_inputs/validation/raw.py [hunk 2]
```diff
@@ -733,13 +785,43 @@ def validate_exposure_standard_deviation(data: pd.DataFrame, entity: Union[RiskF
...
+    """Check the standard set of validations on raw exposure distribution
+    weights data for entity, replacing the age ids check with a custom check
+    for the all ages age group since distribution weights are not age specific.
+    Because exposure distribution weights are neither age nor sex specific
+    (and risk factor age restrictions don't correspond to data), skip all
+    restriction checks for risk factors.
+
+    Additionally, verify that distribution weights sum to 1.
+
+
```
Comment (collijk) [inline src/vivarium_inputs/validation/raw.py]: Extra return

File: src/vivarium_inputs/validation/raw.py [hunk 3]
```diff
@@ -713,6 +741,30 @@ def validate_exposure(data: pd.DataFrame, entity: Union[RiskFactor, CoverageGap,
...
+        Expected set of years, used to check the `year_id` column in `data`.
+
+    Raises
+    ------
+    DataAbnormalError
+        If data does not exist, expected columns are not found in data, or
+        any values in columns do not match the expected set of values.
+
+    """
+    # FIXME: why are we checking data exist on the filtered data? we do that for no other entity/measure
```
[the remark this answers is not in the document]
Comment (michaella21) [reply src/vivarium_inputs/validation/raw.py]: I think it happened after you left on Friday. We talked about that exposure sd should not exist when there is no exposure, also we don't want to raise an error if there's any non-zero data that we don't care much. But I might be wrong, any thoughts @collijk

[the remark this answers is not in the document]
Comment (kfwilson) [reply src/vivarium_inputs/validation/raw.py]: ok that makes sense. I'll add that to the doc then

File: src/vivarium_inputs/validation/raw.py [hunk 4]
```diff
@@ -733,13 +785,43 @@ def validate_exposure_standard_deviation(data: pd.DataFrame, entity: Union[RiskF
 
     check_age_group_ids(data, age_start, age_end)
     check_sex_ids(data, True, True)
-
+    # FIXME: why did we get rid of the sex restriction check?
```
[the remark this answers is not in the document]
Comment (kfwilson) [reply src/vivarium_inputs/validation/raw.py]: ok no problem. adding it back in

**Pull request merged**